src/openMVG/sfm/pipelines/sequential/sequential_SfM.cpp

```diff
@@ -1027,7 +1027,7 @@ bool SequentialSfMReconstructionEngine::Resection(const size_t viewIndex)
     }
     if(!sfm::SfM_Localizer::RefinePose(
```

Comment:
what about "wasting" 2 booleans like that
```c++
const bool b_refine_pose = true;
const bool b_refine_intrinsics = false;
```
and use it instead of explicit `true` and `false`. That way whoever reads, can understand what is happening without having to jump to the definition of `RefinePose`.